keyboards/kudox/columner/info.json

```diff
@@ -4,6 +4,7 @@
     "keyboard_name": "Kudox Keyboard Columner",
     "layouts": {
         "LAYOUT": {
+            "key_count": 66,
```

Comment:
I'm pretty sure you aren't supported have these. 
```suggestion
```

Reply:
Looks like you missed this one.

Reply:
@drashna 
I am sorry! I modify it now... 🙇